src/List/ListItem.js

```diff
@@ -5,8 +5,8 @@ import ColorManipulator from '../utils/colorManipulator';
 import transitions from '../styles/transitions';
 import EnhancedButton from '../internal/EnhancedButton';
 import IconButton from '../IconButton';
-import OpenIcon from '../svg-icons/navigation/arrow-drop-up';
-import CloseIcon from '../svg-icons/navigation/arrow-drop-down';
+import OpenIcon from '../svg-icons/navigation/keyboard-arrow-up';
+import CloseIcon from '../svg-icons/navigation/keyboard-arrow-down';
```

Comment:
Why not using https://github.com/callemall/material-ui/blob/master/src/svg-icons/hardware/keyboard-arrow-down.js?